### PR TITLE
feature: TRACEFOSS-1730 preperation for new api assetAsPlanned

### PR DESCRIPTION
**Pull request by ds-mwesener**

### CODE REVIEW

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/infrastructure/base/irs/model/response/semanticdatamodel/SemanticDataModel.java [hunk 1]
```diff
@@ -71,40 +74,54 @@ public Optional<String> getLocalId(LocalIdKey key) {
                 .map(LocalId::value);
     }
 
+    public List<DetailAspectModel> extractDetailAspectModelsAsPlanned() {
```
Comment (ds-lcapellino) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/assets/infrastructure/base/irs/model/response/semanticdatamodel/SemanticDataModel.java]: this method can be private

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/common/config/AssetsAsyncConfig.java [hunk 2]
```diff
@@ -29,7 +29,7 @@
 public class AssetsAsyncConfig {
 
 	public static final String SYNCHRONIZE_ASSETS_EXECUTOR = "synchronize-assets-executor";
-	public static final String LOAD_SHELL_DESCRIPTORS_EXECUTOR = "load-shell-descriptors-executor";
+    	public static final String LOAD_SHELL_DESCRIPTORS_EXECUTOR = "load-shell-descriptors-executor";
```
Comment (ds-lcapellino) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/common/config/AssetsAsyncConfig.java:32]: Maybe run reformat over some classes

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/application/alert/rest/AlertController.java [hunk 3]
```diff
@@ -128,7 +128,8 @@ public QualityNotificationIdResponse alertAssets(@RequestBody @Valid StartQualit
                 request.getDescription(),
                 request.getTargetDate(),
                 request.getSeverity().toDomain(),
-                request.getBpn()
+                request.getBpn(),
+                request.isAsBuilt()
```
Comment (ds-lcapellino) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/application/alert/rest/AlertController.java:132]: We should refactor this method to just take request as parameter

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/domain/service/NotificationPublisherService.java [hunk 4]
```diff
@@ -23,31 +23,22 @@
...
+import org.eclipse.tractusx.traceability.assets.domain.base.model.AssetBase;
 import org.eclipse.tractusx.traceability.common.model.BPN;
 import org.eclipse.tractusx.traceability.common.properties.TraceabilityProperties;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.model.QualityNotification;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.model.QualityNotificationAffectedPart;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.model.QualityNotificationMessage;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.model.QualityNotificationSeverity;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.model.QualityNotificationSide;
-import org.eclipse.tractusx.traceability.qualitynotification.domain.model.QualityNotificationStatus;
+import org.eclipse.tractusx.traceability.qualitynotification.domain.model.*;
```
Comment (ds-lcapellino) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/domain/service/NotificationPublisherService.java:34]: don't use * as import

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/domain/service/NotificationPublisherService.java [hunk 5]
```diff
@@ -23,31 +23,22 @@
...
 import java.time.Instant;
 import java.time.LocalDateTime;
-import java.util.ArrayList;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```
Comment (ds-lcapellino) [inline tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/domain/service/NotificationPublisherService.java:41]: dont use * as import

File: tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/application/alert/rest/AlertController.java (hunk 3 above)
Comment (ds-mwesener) [reply tx-backend/src/main/java/org/eclipse/tractusx/traceability/qualitynotification/application/alert/rest/AlertController.java:132]: Yes, agree. Added todo comment.

**Review by ds-lcapellino — DISMISSED** [review]
LGTM

**Review by ds-lcapellino — APPROVED** [review]
LGTM

**Pull request merged**